### PR TITLE
[FSDP] fix: fix fsdp llh calc div temperature to match megatron

### PR DESCRIPTION
**Pull request by jeffrey-dot-li**
Megatron LLH calculation divides logits by temperature, this fixes fsdp to match

https://github.com/THUDM/slime/blob/4badb749b213408bdf24d799d6c781b2d3c5c31f/slime/backends/megatron_utils/loss.py#L33

### CODE REVIEW

**Comment by UbeCc** [comment]
Well it is implemented in https://github.com/THUDM/slime/pull/344. Both implementation is ok!

**Comment by jeffrey-dot-li** [comment]
o nice thanks ill close then
